application/client/src/app/schema/render/columns.ts

```diff
@@ -67,6 +85,22 @@ export class Columns {
         return this.headers[column].visible;
     }
 
+    public setColor(column: number, color: string): void {
+        if (this.headers[column] === undefined) {
+            throw new Error(`Maximum ${this.headers.length} present in the file and tried to set the color of column at #${column}`);
+        }
+        this.headers[column].color = (color === 'rgba(0,0,0,0)') ? '#FFF' : color;
```

Comment:
Please do not hardcode values of colors. Use theme/scheme from `@ui/styles/colors`